yt_dlp/extractor/thirtydaysinger.py

```diff
@@ -0,0 +1,104 @@
+import re
+
+from .wistia import WistiaBaseIE
+from ..utils import clean_html, get_elements_html_by_class
+
+
+class ThirtyDaySingerBase(WistiaBaseIE):
```

Comment:
needs to be named `BaseIE`
```suggestion
class ThirtyDaySingerBaseIE(WistiaBaseIE):
```